site/src/pages/release-notes/1.24.3.mdx

```diff
@@ -0,0 +1,26 @@
+---
+date: 2023-07-25
+---
+
+## 🛠️ Bug fixes
+
+- The decorators work correctly even when the request path for a Spring controller contains
```

Comment:
Why don't you explain directly what part has changed in the first line?

```
- Armeria server now strips [matrix variables](...) from a request path before
  finding a matching route.
```

Reply:
That's a good suggestion. 👍